### ISSUE TITLE
css.properties.mask-image - Google Chrome doesn't support <mask>

### ISSUE THREAD

**Issue by lilydjwg** [mdn/browser-compat-data#12888]
#### What information was incorrect, unhelpful, or incomplete?

The note says Google Chrome "implemented with the vendor prefix: -webkit-".

#### What did you expect to see?

The note should also say that Google Chrome doesn't support using a <mask> element in the SVG file, like this:

```css
  -webkit-mask-image: url(abc.svg#mymask);
```

Google Chrome ignores the `#mymask` part and uses the SVG image content.

#### Did you test this? If so, how?

Yes, with Google Chrome 94.0.4606.81. Just try to use <mask> and it works in Firefox but not Google Chrome. What else can I say about the test?

Also there is [a question](https://stackoverflow.com/q/20600608/296473) on stackoveflow about this.

<details>
<summary>MDN page report details</summary>

* Query: `css.properties.mask-image`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/mask-image
* Report started: 2021-10-16T12:00:20.144Z

</details>

**Comment by caugner** [comment]
@lilydjwg I know this is an old issue, but do you remember the test case you used to reproduce the issue in Chrome?

**Comment by lilydjwg** [comment]
No, I can't the test case now. Perhaps it was deleted and replaced by other methods since it didn't work as expected.
